### ISSUE TITLE
In-App update crashes

### ISSUE THREAD

**Issue by stentrup** [codecentric/merge-request-notifier#68]
The following error occurs when I click on "Install & Restart":

```
Error: Please check update first
    at c.downloadUpdate (/Applications/Merge Request Notifier.app/Contents/Resources/app.asar/dist/main.js:1:30806)
    at IpcMain.<anonymous> (/Applications/Merge Request Notifier.app/Contents/Resources/app.asar/dist/main.js:2:327031)
    at IpcMain.emit (events.js:203:13)
    at WebContents.<anonymous> (electron/js2c/browser_init.js:3829:15)
    at WebContents.emit (events.js:203:13)
```

---

merge-request-notifier 1.0.0
Electron 7.1.8
darwin 19.2.0
Locale: de

![image](https://user-images.githubusercontent.com/3942288/72239570-b2afc980-35e1-11ea-97e5-73f2d7a68905.png)

**Comment by ruettenm** [comment]
Will be fixed in the next release

**Comment by ruettenm** [comment]
Fixed in Version **1.0.1**.
